### DISCUSSION TITLE
Лайки/дизлайки

### DISCUSSION THREAD

**Discussion by ililmusic** [Chimildic/YaMuTools#102] [Вопрос/ответ]
Сколько надо ждать чтобы снять все дизлайки? (6000+ дизлайков) За часа 2-3 страница так и не перезагрузилась, а дизлайки все на месте.

**Comment by Chimildic** [comment]
Скорее всего запрос споткнулся об какой-то трек или ЯМ перестал отвечать из-за спама. Повторную команду пробовали?
